### DISCUSSION TITLE
How to make check button?

### DISCUSSION THREAD

**Discussion by yutanov** [python-telegram-bot/python-telegram-bot#2950] [Q&A]
I'd like to make check button changing color and value callback_data when pressed. Is there any forms with check buttons and the send button?

**Comment by Bibo-Joshi** [comment]
hi. It's unclear to me what exactly you're asking for. please have a look at [this guide](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Ask-Right) on how to ask good questions.

**Reply by yutanov** [reply]
Oh, sorry.
It must be something like this at the picture.
![hqHX9](https://user-images.githubusercontent.com/86551782/163025530-53bc688c-9fa5-4031-a518-1cd9e4933230.png)
